handbook/python-dataplugins/python-dataplugins.md

```diff
@@ -0,0 +1,308 @@
+# Python DataPlugins
+
+Create a DataPlugin to load, to register, or to search your own file formats in LabVIEW or DIAdem, or to index and to browse your own file formats with SystemLink DataFinder.

```

Comment:
As the header is "Python DataPlugins", would it be better to start with "Create a Python DataPlugin to load, register, or search your custom file formats in (question is whether the file formats are "in" LV or DD or you search them "with" LV or DD) LabVIEW or DIAdem, or to index, browse, (and find?) your file formats with a SystemLink DataFinder.

Reply:
thanks, your text sounds good to me!